maps/biter_battles_v2/feeding.lua

```diff
@@ -360,9 +360,11 @@ function Public.feed_biters_mixed_from_inventory(player, button)
         'electromagnetic-science-pack',
         'agricultural-science-pack',
         'cryogenic-science-pack',
+        'promethium-science-pack',
     }
     if button == defines.mouse_button_type.right then
         food = {
+			'promethium-science-pack',
```

Comment:
Please use only spaces for indentation.

basic BB also has somewhat strict code style using stylua tool. But I haven't exactly followed it with my space-age fork.

Reply:
should be fixed now!